templates/ibm/index.html

```diff
@@ -0,0 +1,163 @@
+
+{% block content %}
+
+<section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom" style="background-image: url(https://assets.ubuntu.com/v1/3978eb15-cloud-lifestyle-with-screens.png?w=1500); background-position: right bottom; background-size: 1246px auto;">
+  <div class="p-strip--suru-background">
+    <div class="row">
+      <div class="col-6 u-vertically-center">
+        <h1>
+          Ubuntu on IBM
+        </h1>
+        <p>
+          Ubuntu is the operating system of choice for next generation, cloud workloads. Canonical and the Ubuntu community work together with IBM to ensure that Ubuntu works seamlessly across IBM platforms, including IBM Z and LinuxONE, IBM Power Systems, and the IBM Cloud.
+        </p>
+        <p>
+          <a class="p-button--positive js-invoke-modal u-no-margin--bottom" href="/ibm/contact-us">
+            Speak to the experts
+          </a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h2>
+        Secure Execution for Linux
+      </h2>
+      <p>
+        IBM Z delivers a Trusted Execution Environment with Ubuntu 20.04 LTS and new single frame models. Announcing <a href="https://www.ibm.com/marketplace/z15">z15 T02</a>, <a href="https://www.ibm.com/us-en/marketplace/linuxone-iii">LinuxONE III LT2</a> and IBM Secure Execution for Linux - enabling clients to isolate large numbers of workloads with granularity and at scale, within a Trusted Execution Environment.
+      </p>
+      <p>
+        We are actively collaborating with IBM in areas like containers, virtualization, Infrastructure-as-a-Service, big data analytics and DevOps to provide reference architectures, support solutions and cloud offerings, both for enterprise data centers and cloud service providers.
+      </p>
+      <p>
+        See what’s possible with our multi-cloud solutions:
+      </p>
+
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Ubuntu for IBM Z and LinuxONE on premises</li>
+        <li class="p-list__item is-ticked">Ubuntu for IBM Power</li>
+        <li class="p-list__item is-ticked">Private cloud with Ubuntu and IBM Z or LinuxONE</li>
+        <li class="p-list__item is-ticked">Ubuntu on  IBM Public Cloud</li>
+        <li class="p-list__item is-ticked">Ongoing enterprise level support alongside Global Technology Services</li>
+      </ul>
+      <p>
+        <a href="https://people.canonical.com/~fheimes/Ubuntu_Server_s390x.pdf">
+          What’s new in Ubuntu Server for IBM Z and LinuxONE&nbsp;&rsaquo;
+        </a>
+      </p>
+    </div>
+    <div class="col-4 u-hide--small u-align--center">
+      {{  image(      url="https://assets.ubuntu.com/v1/2e3b72d3-2018-logo-IBM.svg",      alt="IBM logo",      width="250",      height="98",      hi_def=True,      loading="lazy") | safe}}
```

Comment:
Some weird formatting here

Reply:
I don't think it matters?  I can change if you want.

Reply:
Updated